f8a_worker/workers/graph_populator.py

```diff
@@ -0,0 +1,242 @@
+                           "pkg.property('gh_prs_last_month_opened', " + cls.str_value_cleaner(gh_prs_last_month_opened) + ");" \
+                           "pkg.property('gh_prs_last_year_closed', " + cls.str_value_cleaner(gh_prs_last_year_closed) + ");" \
+                           "pkg.property('gh_prs_last_month_closed', " + cls.str_value_cleaner(gh_prs_last_month_closed) + ");" \
+                           "pkg.property('gh_issues_last_year_opened', " + cls.str_value_cleaner(gh_issues_last_year_opened) + ");" \
+                           "pkg.property('gh_issues_last_month_opened', " + cls.str_value_cleaner(gh_issues_last_month_opened) + ");" \
+                           "pkg.property('gh_issues_last_year_closed', " + cls.str_value_cleaner(gh_issues_last_year_closed) + ");" \
+                           "pkg.property('gh_issues_last_month_closed', " + cls.str_value_cleaner(gh_issues_last_month_closed) + ");" \
+                           "pkg.property('gh_forks', " + cls.str_value_cleaner(gh_forks) + ");" \
+                           "pkg.property('gh_stargazers', " + cls.str_value_cleaner(gh_stargazers) + ");" \
+                           "pkg.property('gh_open_issues_count', " + cls.str_value_cleaner(gh_open_issues_count) + ");" \
+                           "pkg.property('gh_subscribers_count', " + cls.str_value_cleaner(gh_subscribers_count) + ");"
+
+        # Add tokens for a package
+        for tkn in pkg_name_tokens:
+            if tkn:
+                str_package += "pkg.property('tokens', '" + cls.str_value_cleaner(tkn) + "');"
+
+        # Get Libraries.io data
+        if 'libraries_io' in input_json.get('analyses', {}):
+            libio_dependents_projects = input_json.get('analyses').get('libraries_io').get('details', {}) \
+                                                  .get('dependents', {}).get('count', -1)
+            libio_dependents_repos = input_json.get('analyses').get('libraries_io').get('details', {}) \
+                                               .get('dependent_repositories', {}).get('count', -1)
+            libio_total_releases = input_json.get('analyses').get('libraries_io').get('details', {}) \
+                                             .get('releases', {}).get('count', -1)
+            libio_latest_release = input_json.get('analyses').get('libraries_io').get('details', {}) \
+                                             .get('releases', {}).get('latest', {}).get('published_at')
+            libio_latest_version = input_json.get('analyses').get('libraries_io').get('details', {}) \
+                                             .get('releases', {}).get('latest', {}).get('version', '')
+
+            if libio_latest_release is not None:
+                try:
+                    prp_package += "pkg.property('libio_latest_release', '" + \
+                                   str(time.mktime(datetime.strptime(libio_latest_release,
```

Comment:
Why is this date magic is required here?

Reply:
converts the date into required format

Reply:
so it converts date from some format to format which is defined in a schema of the graph. Is that correct?

Reply:
👍